### ISSUE TITLE
Upgrade to Octobercms v1.1

### ISSUE THREAD

**Issue by pablorq** [aspendigital/docker-octobercms#43]
Hi,

Is there any way or plans to upgrade the docker container to october v1.1?

More info in the [release notes](https://octobercms.com/support/articles/release-notes).

**Comment by petehalverson** [comment]
Yes, we are planning to upgrade this image to support v1.1 in the near future.

**Comment by pablorq** [comment]
@petehalverson is there any (easy) way to create my own docker image with octobercms v1.1?

I've took a look to dockerfile.template but this seems to be prepared to work with update.sh script.

Thank you in advance.
